cpp/include/cuspatial/cubic_spline.hpp

```diff
@@ -84,4 +84,36 @@ std::unique_ptr<cudf::table> cubicspline_coefficients(
   cudf::column_view const& ids,
   cudf::column_view const& offsets,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
+
+namespace detail {
+
+/**
+ * @brief Finds the lower interpolant position of query_points from a set of
+ * interpolation independent variables.
+ *
+ * @param[in] query_points column of coordinate values to be interpolated.
+ * @param[in] spline_ids ids that identify the spline to interpolate each
+ * coordinate into.
+ * @param[in] offsets int32 column of offsets of the source_points.
+ * This is used to calculate which values from the coefficients are
+ * used for each interpolation.
+ * @param[in] source_points column of the original `t` values used
```

Comment:
We don't need the `[in]` on these. All of our parameters are input, and the output is returned.